ziti/cmd/ascode/importer/importer.go

```diff
@@ -111,7 +90,7 @@ func NewImportCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	return cmd
 }
 
-func (importer *Importer) Init() {
+func (importer *Importer) Execute(input []string) (map[string]any, error) {
```

Comment:
i'd probably change this to take named args since it's a function. passing a slice is a bit strange imo